modules/apps/frontend-js/frontend-js-web/src/main/resources/META-INF/resources/liferay/modal/components/SimpleInputModal.es.js

```diff
@@ -63,6 +63,18 @@ const SimpleInputModal = ({
 	const _handleSubmit = (event) => {
 		event.preventDefault();
 
+		const error = inputValue
+			? ''
+			: Liferay.Language.get('this-field-is-required');
+
+		if (error && required) {
+			setErrorMessage(error);
```

Comment:
It'a a bit awkward that the same UI location for error message is also used for response error. Typical behavior in DXP is for response error is to open error toast, and not show it as form feedback. But, I guess that's out of the scope for this task.